src/Entity.php

```diff
@@ -88,11 +85,11 @@ protected function snakeCase($input)
      *
      * @return Entity
      */
-    protected function request($method, $relativeUrl, $data = null, $additionHeader = array())
+    protected function request($method, $relativeUrl, $data = null)
     {
         $request = new Request();
 
-        $response = $request->request($method, $relativeUrl, $data, $additionHeader);
```

Comment:
@neera11 , Please make sure you revert all the changes related to this from last PR merged.

Reply:
yes i reverted all changes